packages/webview/dist/esm/web.js

```diff
@@ -0,0 +1,6 @@
+import { WebPlugin } from '@capacitor/core';
+export class WebviewBackgroundWeb extends WebPlugin {
+    async changeBackgroundColor(options) {
+        console.log('Color changed: ', options.color);
```

Comment:
Looks like this log was removed [as requested](https://github.com/cybersemics/em/pull/2594#discussion_r1859023507) from the source (`packages/webview/src/web.ts`) but then not rebuilt and included in the build here.

One example of why we should build from source rather than including the `dist` in the repo :)